### PR TITLE
dtspo-1835, move to templates v2

### PR DESCRIPTION
**Pull request by hannah38**
Updated to latest library
Updated to templates v2
Readme updated, added note about why language is set to none
Tested with plum on preview, https://github.com/hmcts/cnp-plum-recipes-service/pull/329
`plum-recipe-backend-pr-329-java-7f798f6d4b-jr6nm   1/1     Running            0          11m`

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +61,51 @@ keyVaults:
...
+  java:
+    replicas: 2
+    memoryRequests: '1024Mi'
+  ```
+  will set replicas to `2` , **but memoryRequests will still be `512Mi`**. It can be set `1024Mi` by using below config
+
+  ```yaml
+    java:
+      replicas: 2
+      java:
```
Comment (timja) [inline README.md:105]: image

java java

Comment (hannah38) [reply README.md]: Corrected

File: README.md [hunk 2]
```diff
@@ -8,6 +8,15 @@ We will take small PRs and small features to this chart but more complicated nee
...
 
+This chart adds below templates from [chart-library](https://github.com/hmcts/chart-library/) based on the chosen configuration:
+
+- [Deployment](https://github.com/hmcts/chart-library/tree/master#deployment)
+- [Horizontal Pod Auto Scaler](https://github.com/hmcts/chart-library/tree/master#hpa-horizontal-pod-auto-scaler)
+- [Ingress](https://github.com/hmcts/chart-library/tree/master#ingress)
+- [Pod Disruption Budget](https://github.com/hmcts/chart-library/tree/master#pod-disruption-budget)
+- [Service](https://github.com/hmcts/chart-library/tree/master#service)
+- [Deployment Tests](https://github.com/hmcts/chart-library/tree/master#smoke-and-functional-tests)
+
```
Comment (adusumillipraveen) [inline README.md:19]: Might add a reference to keyvaults secrets and refer it for people coming here for secrets https://github.com/hmcts/chart-library#keyvault-secret-csi-volumes

Comment (hannah38) [reply README.md:20]: Added

**Pull request merged**